vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -698,12 +698,15 @@ base_learner* ccb_explore_adf_setup(options_i& options, vw& all)
   data->id_namespace_str.append("_id");
   data->id_namespace_hash = VW::hash_space(all, data->id_namespace_str);
 
-  learner<ccb, multi_ex>& l = init_learner(data, base, learn_or_predict<true>, learn_or_predict<false>, 1,
```

Comment:
where is the 1 specified now?

Reply:
1 is the default and so it doesnt need to be added in that case